lib/SILOptimizer/Mandatory/TFLowerGraph.cpp

```diff
@@ -307,13 +312,18 @@ struct TFGraphFunctionLowering
   /// Note: `name` starts with _, where the name of a graph node cannot start
   /// with _.
   ///
-  /// This sets hasSideEffects to true if the body of the graph had any side
-  /// effecting operations.  This allows calls to the graph to set up control
-  /// dependence edges properly.
+  /// `funcHasSideEffects` is set to true if the body of the graph function has
+  /// any side effecting operations.  This allows the caller to this graph
+  /// function to wire up control dependence edges properly. In particular, the
+  /// graph node that runs this function in the call-site is treated as a
+  /// side-effecting node, and will be threaded via control edges with other
+  /// side-effecting nodes in the caller. This is unless we are building a top
```

Comment:
"This is _true_ unless"?

Reply:
I feel saying "This is unless" here flows a bit better -- is it not grammatically correct?

Reply:
"Unless" is a _conjunction_ and cannot directly follow a verb.

Reply:
k, fixed. Thanks!